docs/08-cicd.md

```diff
@@ -10,10 +10,10 @@
 
 ## Triggering a Delivery using the CI
 
-The easiest way to deliver a new build to production or development is to trigger the corresponding CircleCI workflow *api_triggered_delivery*.
+The easiest way to deliver a new build to production or development is to trigger the corresponding CircleCI workflow *api_triggered_delivery*:
 
 * Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-pi-tokens/).
-* Trigger a build using the tool [trigger-pipeline.sh](../.circleci/trigger-pipeline.sh).
+* Trigger a build using the tool [trigger-pipeline.sh](.circleci/trigger-pipeline).
```

Comment:
Does this trigger a delivery or only a build? Of the current origin develop branch? Maybe add this here if applicable.